src/evidently/ui/api/static.py

```diff
@@ -36,7 +36,6 @@ async def index() -> Response:
     path="",
     route_handlers=[
         index,
-        create_static_files_router("/static", directories=[pathlib.Path(base_path) / "static"]),
-        create_static_files_router("/", directories=[pathlib.Path(base_path) / "assets"]),
+        create_static_files_router("/", directories=[pathlib.Path(base_path)]),
```

Comment:
```suggestion
        create_static_files_router("/", directories=[base_path]),
```

This can be just that :)